android/src/main/kotlin/io/linkrunner/flutter/LinkrunnerPlugin.kt

```
@@ -126,12 +128,32 @@ class LinkrunnerPlugin: FlutterPlugin, MethodCallHandler {
                 val linkRunner = NativeLinkRunner.getInstance()
                 nativeLinkRunner = linkRunner
                 
+                // Configure SDK with client platform and version prior to init
+                try {
+                    if (packageVersion != null) {
+                        NativeLinkRunner.configureSDK(platform, packageVersion)
+                    } else {
+                        // fallback when version is unavailable
+                        NativeLinkRunner.configureSDK(platform, "unknown")
+                    }
+                } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)